src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,33 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
```

Comment:
Good job Ali! for that line you don't have to add it because the grid will automatically fold to 1 column when it is below 488px :)